sys/include/crypto/aes.h

```diff
@@ -46,20 +46,35 @@ typedef uint8_t u8;
 
 #define AES_MAXNR         14
 #define AES_BLOCK_SIZE    16
-#define AES_KEY_SIZE      16
+
+/**
+ * @name AES key sizes
+ *
+ * @{
+ */
+#define AES_KEY_SIZE_128    16
+#define AES_KEY_SIZE_192    24
+#define AES_KEY_SIZE_256    32
+/** @} */
+
+/**
+ * @brief AES key size default config
+ */
+#if !IS_USED(MODULE_CRYPTO_AES_128) && !IS_USED(MODULE_CRYPTO_AES_192) && \
+    !IS_USED(MODULE_CRYPTO_AES_256)
+    #define MODULE_CRYPTO_AES_128 1
+#endif
```

Comment:
The `MODULE_` macros are injected by the build system. It would make sense to raise an `#error` under the condition where no module is being used.